bdb/file.c

```diff
@@ -7622,26 +7630,27 @@ int bdb_have_unused_files(void) { return oldfile_list_empty() != 1; }
 int bdb_purge_unused_files(bdb_state_type *bdb_state, tran_type *tran,
                            int *bdberr)
 {
-    char *munged_name = NULL;
-    int rc;
-    unsigned lognum = 0, lowfilenum = 0;
+  char *munged_name = NULL;
```

Comment:
I did indent by 4 spaces.  The instructions to fix the failure reindented to code to 2 space.